.gitignore

```diff
@@ -396,4 +396,8 @@ FodyWeavers.xsd
 
 # JetBrains Rider
 *.sln.iml
-/.idea
\ No newline at end of file
+/.idea
+
+# DocFX
+_site/
+api/
```

Comment:
When the Docfx dotnet tool is run locally, the folders `_site` and `api/` get created with the generated documentation. This was added to ignore these files from future development and testing.